Expensify/react-native-live-markdown
# Introduce autoFocus functionality for web

### Details
This PR introduces useEffect for handling `autoFocus` of the contenteditable div

### Related Issues
#174 

### Manual Tests
1. set `autoFocus` to true in example app
2. Open web example
3. refresh page and verify that input is being auto focused

### Linked PRs
